test-suite/squidconf/time_units

```diff
@@ -34,15 +33,15 @@ max_stale 1 second
 max_stale 0.0167 minute
 
 # maximum checks
```

Comment:
Same concerns apply to the max_stale section. To avoid duplicating our explanation (whatever it ends up being), I suggest:

```suggestion
# maximum checks (see similar dns_timeout checks for details)
```